apps/src/applab/designElements/ZOrderRow.jsx

```diff
@@ -19,13 +19,23 @@ var ZOrderRow = React.createClass({
 
   render: function() {
     var element = this.props.element;
-    var index = Array.prototype.indexOf.call(element.parentNode.children, element);
+
+    // Element will be wrapped in a resizable div
+    var outerElement = element.parentNode;
+    var index = Array.prototype.indexOf.call(outerElement.parentNode.children, outerElement);
     var isBackMost = index === 0;
-    var isFrontMost = index + 1 === element.parentNode.children.length;
+    var isFrontMost = index + 1 === outerElement.parentNode.children.length;
 
     var squareButton = {
       width: 42,
-      height: 42
+      height: 42,
+      backgroundColor: '#0094ca' // $cyan
```

Comment:
I thought about that. I added the comment so that we would at least see this usage if we search for it. Ultimately, I'd prefer that our react stuff be all one way or another if possible (rather than sometimes using classes and sometimes using inline styles)